yarn-project/aztec.js/src/utils/field_compressed_string.ts

```diff
@@ -0,0 +1,27 @@
+import { Fr } from '@aztec/circuits.js';
+
+/**
+ * The representation of a FieldCompressedString in aztec.nr
+ */
+interface NoirFieldCompressedString {
+  /**
+   * The field value of the string
+   */
+  value: bigint;
+}
+/**
+ * This turns
+ * @param field - The field that contains the string
+ * @returns - the string that is decoded from the field
+ */
+export const readFieldCompressedString = (field: NoirFieldCompressedString): string => {
```

Comment:
this can be used in our e2e tests and in unit tests etc - can you check where relevant?